docs/howto/plugins/apis.md

```diff
@@ -93,5 +92,39 @@ If you run NOMAD with this plugin following our [Oasis configuration documentati
 you can curl this API and should receive the message:
 
 ```sh
-curl localhost/nomad-oasis/myapi
+curl localhost:8000/nomad-oasis/myapi/
+```
+
+In a nomad-distro-dev environment, the API should be accessible at:
+
+```sh
+curl localhost:8000/fairdi/nomad/latest/myapi/
+```
```

Comment:
Instead of having to account for these two options always in our docs, we should just make it so that both `nomad-distro-template` and `nomad-distro-dev` use the same path by default. I have created an issue here: https://github.com/FAIRmat-NFDI/nomad-distro-dev/issues/75.